lib/graphql/query/arguments.rb

```diff
@@ -85,7 +95,7 @@ def each_value
         end
       end
 
-      NO_ARGS = self.new({}, argument_definitions: [])
+      NO_ARGS = self.new({}, argument_definitions: [], defaults_used: {})
```

Comment:
Should be `Set.new`? I guess it's splitting hairs since `#include?` will still return false..